src/Symfony/Component/Intl/Data/Generator/TimezoneDataGenerator.php

```diff
@@ -121,8 +121,6 @@ protected function generateDataForRoot(BundleEntryReaderInterface $reader, strin
 
     protected function generateDataForMeta(BundleEntryReaderInterface $reader, string $tempDir): ?array
     {
-        $rootBundle = $reader->read($tempDir, 'root');
```

Comment:
`$reader->read()` can throw IIRC

Reply:
You're right !

Reply:
Indeed some implementations can throw `Symfony\Component\Intl\Exception\ResourceBundleNotFoundException` or `Symfony\Component\Intl\Exception\RuntimeException`

Can we add a `@throws ResourceBundleNotFoundException` on `BundleReaderInterface` ?

Reply:
I reverted this part